config/v1/types_infrastructure.go

```diff
@@ -231,22 +231,22 @@ const (
 	IBMCloudProviderTypeUPI IBMCloudProviderType = "UPI"
 )
 
-// ClusterHostedDNS indicates whether the cluster DNS is hosted by the cluster or Core DNS .
```

Comment:
These types were introduced in 4.15 under techpreview, our intention now is to change the structure around so dropping this old field. Also intending to backport the change to 4.15 before it ships so this version of the API should never see the light of day